README.md

```diff
@@ -37,6 +37,19 @@ Open up [localhost:3000](http://localhost:3000) and start clicking around.
 
 Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
 
+### Running tests
+
+Functional tests are located in the [`cypress/integration`](cypress/integration) folder. Prior to running tests, you'll need to install [cypress](https://www.cypress.io/) globally:
+```bash
+npm i -g cypress
+```
+
+Then you can run the tests using the following commands :
+```bash
+npm t
+npm run cy:run
+npm run cy:open
```

Comment:
These 3 lines can be combined to:

```bash
npm test
```

`test` [runs `cy:run`](https://github.com/sveltejs/sapper-template/blob/db19dc093ab38ff9a0ad234f444ce1683c96548d/package_template.json#L12).

`cy:open` opens the [interactive test runner](https://docs.cypress.io/guides/guides/command-line.html#cypress-open) which is not necessary. We may want to document it but I think that should be a new paragraph if we do.